ArduPlane/servos.cpp

```diff
@@ -649,7 +649,14 @@ void Plane::servos_twin_engine_mix(void)
 {
     float throttle = SRV_Channels::get_output_scaled(SRV_Channel::k_throttle);
     float rud_gain = float(plane.g2.rudd_dt_gain) / 100;
-    float rudder = rud_gain * SRV_Channels::get_output_scaled(SRV_Channel::k_rudder) / float(SERVO_MAX);
+
+    // Apply differential throttle proportional to rudder input only if throttle is above the deadzone.
+    // This prevents props from spinning when disarming via RC.
+    float rudder = 0;
+    if (channel_throttle->get_control_in() > 0) {
+        rudder = rud_gain * SRV_Channels::get_output_scaled(SRV_Channel::k_rudder) / float(SERVO_MAX);
```

Comment:
This is the wrong value to check. The stick input is relevent in modes like FBWA, but this function also works in AUTO, CRUISE and other modes where the throttle stick may be at zero while at high thrust.
I think what we need is that if the local throttle variable is zero then we should not do differential thrust